dev/docker/hive/Dockerfile

```diff
@@ -97,54 +122,54 @@ RUN echo "HADOOP_CONF_DIR=${HADOOP_CONF_DIR}" >> /etc/environment
 RUN echo "HADOOP_CLASSPATH=${JAVA_HOME}/lib/tools.jar" >> /etc/environment
 RUN echo "YARN_HOME=${YARN_HOME}" >> /etc/environment
 RUN echo "HIVE_HOME=${HIVE_HOME}" >> /etc/environment
+RUN echo "ZK_HOME=${ZK_HOME}" >> /etc/environment
 RUN echo "PATH=${PATH}" >> /etc/environment
 RUN echo "CLASSPATH=${CLASSPATH}" >> /etc/environment
 RUN echo "LD_LIBRARY_PATH=${LD_LIBRARY_PATH}" >> /etc/environment
 
 ################################################################################
-# install hadoop
-RUN mkdir ${HADOOP_HOME}
-ADD packages/hadoop-${HADOOP_VERSION}.tar.gz /opt/
-RUN ln -s /opt/hadoop-${HADOOP_VERSION}/* ${HADOOP_HOME}
-
-# replace configuration templates
-RUN rm -f ${HADOOP_CONF_DIR}/core-site.xml
-RUN rm -f ${HADOOP_CONF_DIR}/hadoop-env.sh
-RUN rm -f ${HADOOP_CONF_DIR}/hdfs-site.xml
-RUN rm -f ${HADOOP_CONF_DIR}/mapred-site.xml
-
-ADD core-site.xml ${HADOOP_CONF_DIR}/core-site.xml
-ADD hadoop-env.sh ${HADOOP_CONF_DIR}/hadoop-env.sh
-ADD hdfs-site.xml ${HADOOP_CONF_DIR}/hdfs-site.xml
-ADD mapred-site.xml ${HADOOP_CONF_DIR}/mapred-site.xml
-ADD check-status.sh /tmp/check-status.sh
+# install zookeeper
+ADD packages/zookeeper-${ZOOKEEPER_VERSION}.tar.gz /opt/
+RUN ln -s /opt/zookeeper-${ZOOKEEPER_VERSION} ${ZK_HOME}
+RUN mkdir $ZK_HOME/data
```

Comment:
@unknowntpo Please change to `${ZK_HOME}`

Reply:
done